slides/12_Remix.html

```diff
@@ -24,6 +24,727 @@
+              For deployment to common platforms consider using a template (see
+              <span class="code">README.md</span>)
+            </div>
+          </section>
+          <section>
+            <p><span class="code">npm run typecheck</span></p>
+            <p>
+              Runs TypeScript compiler to ensure type-safety in build pipeline
+              (<em>not</em>
+              done automatically by Remix compiler)
+            </p>
+          </section>
+        </section>
+
+        <section class="subtitle-slide">
+          <h1>Routing</h1>
+        </section>
+
+        <section>
+          <section>
+            <h3>Intro to Route File Naming</h3>
+          </section>
+          <section>
+            <h4>General conventions</h4>
+            <ul>
+              <li>
+                Route files are placed in <span class="code">app/routes</span>
+              </li>
+              <li>Flat file structure (no nested route folders)</li>
+              <li>Co-location of non-route files is possible</li>
+            </ul>
+            <aside class="notes">
+              <ul>
+                Flat file structure was introduced as new convention (initially
+                also used nested route folders). Legacy file structure will not
+                be supported anymore in Remix v2.
+              </ul>
+            </aside>
+          </section>
+          <section data-auto-animate>
+            <pre><code class="hljs no-highlight" data-trim data-line-numbers="1" data-id="code"><script type="text/template">
+app/routes/
+    ├── _index.tsx
+    ├── about.tsx
+    ├── pricing.tsx
+    ├── blog._index.tsx
+    ├── blog.$slug.tsx
+    ├── blog.archive.tsx
+    └── blog.authors.tsx
+              </script></code></pre>
+          </section>
+          <section data-auto-animate>
+            <p><span class="code">_index</span> indicates an index route</p>
+            <pre><code class="hljs no-highlight" data-trim data-line-numbers="2,5" data-id="code"><script type="text/template">
+app/routes/
+    ├── _index.tsx
+    ├── about.tsx
+    ├── pricing.tsx
+    ├── blog._index.tsx
+    ├── blog.$slug.tsx
+    ├── blog.archive.tsx
+    └── blog.authors.tsx
+              </script></code></pre>
+            <p>
+              &rarr; URLs <span class="code">/</span> and
+              <span class="code">/blog</span>
+            </p>
+          </section>
+          <section data-auto-animate>
+            <p>Static route segments</p>
+            <pre><code class="hljs no-highlight" data-trim data-line-numbers="3,4,7,8" data-id="code"><script type="text/template">
+app/routes/
+    ├── _index.tsx
+    ├── about.tsx
+    ├── pricing.tsx
+    ├── blog._index.tsx
+    ├── blog.$slug.tsx
+    ├── blog.archive.tsx
+    └── blog.authors.tsx
+              </script></code></pre>
+            <p>
+              &rarr; e.g.
+              <span class="code">/about</span> and<span class="code"
+                >/blog/archive</span
```

Comment:
**Suggestion:** I think we should add also an explanation on the layout nesting with an example. Here we could add two routes: blog.tsx, and a blog.$slug.comments.tsx as an example, illustrate what each component might include as markup/logic and what will be rendered with which path.

Reply:
I was thinking since this is the same as in React Router, we don't go into detail on this here. Also I am a bit concerned that the chapter is already quite large, so I wanted to keep it lean.
I added a note now to the "General conventions" that the concepts are the same as in React Router and I highlighted the "nested routes" term on the next slide.

Let's see how it is received this time, or if we need to go more into detail in future.